path_login_test.go

```diff
@@ -758,6 +774,35 @@ func TestLoginSvcAcctAndNamespaceSplats(t *testing.T) {
 	}
 }
 
+func TestLoginSvcAcctNamespaceSelector(t *testing.T) {
```

Comment:
I don't see the test for failed login, I wonder if we could turn this into a table driven testcase that tests both cases.